distribution/docker/src/docker/Dockerfile

```diff
@@ -169,15 +168,10 @@ RUN set -e ; \\
     rm "\${TINI_BIN}.sha256sum" ; \\
     mv "\${TINI_BIN}" /rootfs/bin/tini ; \\
     chmod +x /rootfs/bin/tini ; \\
-    curl --retry 10 -L -o /rootfs/bin/busybox \\
-      "https://busybox.net/downloads/binaries/\${BUSYBOX_VERSION}-defconfig-multiarch-musl/busybox-\${BUSYBOX_ARCH}" ; \\
-    chmod +x /rootfs/bin/busybox
-
-# Add links for most of the Busybox utilities
-RUN set -e ; \\
-    for path in \$( /rootfs/bin/busybox --list-full | grep -v bin/sh | grep -v telnet | grep -v unzip); do \\
```

Comment:
I was filtering the list of names that you could invoke busybox with. Besides `telnet`, was that uncessary?

Reply:
I'm not sure what the intent was to begin with. `unzip` is actually required and we even check for it explicitly in our packaging tests. As for telnet, sure, we could delete the symlink, but the capability is built into the `busybox` binary so anyone could just add it back.